hw/top_earlgrey/data/top_earlgrey.sv.tpl

```diff
@@ -354,14 +355,15 @@ module top_${top["name"]} #(
     .wmask_o  (${m["name"]}_wmask),
     .rdata_i  (${m["name"]}_rdata),
     .rvalid_i (${m["name"]}_rvalid),
-    .rerror_i (2'b00)
+    .rerror_i (${m["name"]}_rerror)
   );
 
   ## TODO: Instantiate ram_1p model using RAMGEN (currently not available)
```

Comment:
@eunchan is this comment actually up-to-date? (I have actually no idea what is meant by RAMGEN.)

Reply:
Maybe this refers to the actual SRAM model that we generate for ASIC?
The comment can probably be deleted, since we intend to swap in SRAMs using the primitive wrappers (and we need to do that for all RAMs in the design)...

Reply:
Yeah, that's what I thought as well.

Reply:
I think this can be removed then, as it implicitly applies to all RAMs in the system.

Reply:
Moved the discussion into #2306 to not forget about it.